recipes/sbp/all/test_package/test_package.c

```diff
@@ -0,0 +1,10 @@
+#include <stdio.h>
+#include <string.h>
+
+#include <libsbp/version.h>
+
+int main(void) {
+  fprintf(stderr, "Compiled with sbp %s.\n", SBP_VERSION);
```

Comment:
You're not calling a function of libbsp, just printing a macro.

Calling some functions without having to import `libserialport.h` would be fine.
See https://github.com/swift-nav/libsbp/blob/master/c/examples/example/example.c

Reply:
This comment can be marked as resolved.